src/OSPSuite.Presentation/Presenters/Charts/ChartEditorPresenter.cs

```diff
@@ -26,8 +26,7 @@ namespace OSPSuite.Presentation.Presenters.Charts
    ///    - ChartOptions for editing properties of chartSettings.
    /// </summary>
    public interface IChartEditorPresenter : IPresenter<IChartEditorView>,
-      IListener<ChartUpdatedEvent>,
-      IListener<ChartPropertiesChangedEvent>
```

Comment:
This is not required in this component. The event is only raise from this presenter so we can call the handle directly